core/src/main/java/nl/nn/adapterframework/doc/model/AncestorChildNavigation.java

```diff
@@ -0,0 +1,127 @@
+/* 
+Copyright 2020 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
+package nl.nn.adapterframework.doc.model;
+
+import static nl.nn.adapterframework.doc.model.ElementChild.ALL;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+abstract class AncestorChildNavigation<K, T extends ElementChild> {
```

Comment:
Uit de private methods declaredGroupOrRepeatedChildren() in deze class maak ik op dat deze bedoeld is voor gebruik bij het opbouwen van de Xsd. Klopt dat? Dan zou dat ook in de naam naar voren moeten komen.

Reply:
Gedaan.